packages/grid/_modules_/grid/hooks/utils/useResizeContainer.ts

```diff
@@ -46,7 +46,7 @@ export function useResizeContainer(apiRef): (size: ElementSize) => void {
             'The grid displays with a width of 0px.',
             '',
             'You can find a solution in the docs:',
-            'https://material-ui.com/components/data-grid/rendering/#layout',
+            'https://material-ui.com/components/data-grid/layout',
```

Comment:
URLs should have a trailing slash (`<link rel="canonical" href="https://material-ui.com/components/data-grid/layout/"/>`)

```suggestion
            'https://material-ui.com/components/data-grid/layout/',
```

Reply:
But in the core, we never put hard-coded URLs like this in the source. We use redirections: https://github.com/mui-org/material-ui/blob/8cc7dd0568adde2b8c23804fabdadbd7cfff21d6/docs/public/_redirects#L15

Reply:
> Why, because this is a 404 for developers that have a previous version of the library installed (high version fragmentation)

I didn't get this. So suppose if we put in `_redirects`:
```
/r/data-grid/layout https://material-ui.com/components/data-grid/layout/
```
And in code:
```js
'You can find a solution in the docs:'
'https://material-ui.com/components/r/data-grid/layout',
```
Does this mean that if developers are in the old version, they will get warning in console with this new link `https://material-ui.com/components/r/data-grid/layout'`? How come? If they are using old version won't it will still show the old 404 link `data-grid/rendering/#layout`?